Automattic/vip-go-mu-plugins
# Search Dev Tools: add class name to CollapsibleList Component

## Description
For the search e2e tests

## Changelog Description

### Plugin Updated: Search Dev Tools

Search Dev Tools CollapsibleList component now has the title as part of the className

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Enable ES
2) Go to a search page via `?s=test`
3) Click Search Dev Tools
4) Inspect and look for the added classes on `WP_Query` and `Trace`...it should be `wp_query` for WP_Query and `trace` for Trace
